<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QTabWidget" name="scriptTabs">
          <property name="tabShape">
           <enum>QTabWidget::Triangular</enum>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="documentMode">
           <bool>false</bool>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="text">
             <string>Save Script</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="runButton">
            <property name="text">
             <string>Run Script</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="runSelectedButton">
            <property name="text">
             <string>Run Highlighted (F9)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="terminalWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLoad_Script">
     <property name="title">
      <string>Load Script</string>
     </property>
     <addaction name="actionFrom_Window"/>
     <addaction name="actionFrom_File"/>
    </widget>
    <widget class="QMenu" name="menuRecentScripts">
     <property name="title">
      <string>Recent Scripts</string>
     </property>
     <addaction name="actionNo_Recent_Scripts"/>
    </widget>
    <addaction name="actionNew_Script"/>
    <addaction name="menuRecentScripts"/>
    <addaction name="menuLoad_Script"/>
    <addaction name="actionSave_Script"/>
    <addaction name="actionChangeFontSize"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Script">
   <property name="text">
    <string>New Script</string>
   </property>
  </action>
  <action name="actionSave_Script">
   <property name="text">
    <string>Save Script</string>
   </property>
  </action>
  <action name="actionFrom_Window">
   <property name="text">
    <string>From Window</string>
   </property>
  </action>
  <action name="actionFrom_File">
   <property name="text">
    <string>From File</string>
   </property>
  </action>
  <action name="actionNo_Recent_Scripts">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>No Recent Scripts</string>
   </property>
  </action>
  <action name="actionChangeFontSize">
   <property name="text">
    <string>Change Font Size</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
